src/DFTK.jl

```diff
@@ -13,9 +13,6 @@ using spglib_jll
 using Unitful
 using UnitfulAtomic
 using ForwardDiff
-using AbstractFFTs
-using GPUArraysCore
```

Comment:
Not sure about removing these ... sorry I was only talking about the `using CUDA`.